mailet/base/src/main/java/org/apache/mailet/base/MailetUtil.java

```diff
@@ -109,4 +122,90 @@ public static boolean getInitParameter(MailetConfig config, String name, boolean
         }
         return result;
     }
+
+    public static boolean canSendAutomaticResponse(Mail mail) throws MessagingException {
```

Comment:
bad naming and probably badly located, too. How many classes will need to know if you can auto-reply to an email ?